airqo-platform/AirQo-frontend
# [NetManager] Device monitor refactoring

#### Summary of Changes (What does this PR do?)
- Fixes device management page
- Split overview components
- Pending (maintenace logs and components) on device overview

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Set this backend [PR](https://github.com/airqo-platform/AirQo-api/pull/494)
* Fetch and checkout to this branch locally
* Set the `REACT_APP_BASE_DEVICE_MONITORING_URL` in frontend `.env`. sample value `http://localhost:4001/api/v1/` assuming nothing changed
* Install requirements `npm install`
* Start netmanager application `npm run stage-mac` (macOS) or `npm run stage-pc` (windows Platform)
* Confirm the summary of changes listed above

#### What are the relevant tickets?
- [PLAT-680](https://airqoteam.atlassian.net/browse/PLAT-680)
